pkg/logger/event/event.go

```diff
@@ -26,6 +26,7 @@ const (
 	logType               = "event"
 	TraceIDKey            = "X-Ab-TraceID"
 	SessionIDKey          = "X-Ab-SessionID"
+	FlightIDKey           = "X-Flight-ID"
```

Comment:
the actual header name is `x-flight-id`, all in lowercase. Although `req.HeaderParameter()` is case-insensitive, I think it's better to keep consistency across codebases

Reply:
well noted, thanks for the feedback